Tasks/AzureDevOpsTestPlanV0/automatedTests.ts

```diff
@@ -0,0 +1,53 @@
+import tl = require('azure-pipelines-task-lib/task');
+import { testInvoker } from './automatedTestInvoker'
+import { TestPlanData, getAutomatedTestData } from './getAutomatedTests';
+import { TestCase } from 'azure-devops-node-api/interfaces/TestPlanInterfaces';
+
+
+export async function automatedTestsFlow() {
+
+    let ListOfTestsToBeExecuted: string[] = [];
+
+    console.log(tl.loc('automatedTestsTriggered'));
+    await getFQNsOfAutomatedTestCases()
+        .then((testsToBeExecuted) => {
+            ListOfTestsToBeExecuted = testsToBeExecuted;
+        })
+        .catch((error) => {
+            console.error("Promise rejected:", error);
```

Comment:
Suggestion: Update the error message to reflect the failure, something like 'Error while fetching FqnsOfAutomatedTestCases : error' maybe.

I see this at couple of other places, we can update them as well.

Reply:
updated!